p2p/src/lib.rs

```diff
@@ -150,3 +156,15 @@ pub(crate) mod unbounded_with_len {
         }
     }
 }
+
+/// Create Blake2b hash as u64 value
+pub fn blake2b_hash(slice: impl AsRef<[u8]>) -> u64 {
```

Comment:
I'm not sure should i place this in `iroha_crypto` because it's used only in `iroha_p2p` crate.

Reply:
the only difference is hash length? The proper thing to do would be to make `iroha_crypto::Hash` parametrized with hash length. Which could be a very large change through the codebase, ergo I think you should leave this here

Reply:
The difference is that returned value is ’u64’. I initially want to do arithmetic operations on hashes, but currently i'm using only comparison. So i think i can switch to ’Hash’ directly. But with ’u64’ we still save quite some place. 
What do you think?

Reply:
Let's leave as is.